apps/api-gateway/src/geo-location/geo-location.module.ts

```diff
@@ -0,0 +1,26 @@
+import { Logger, Module } from '@nestjs/common';
+import { GeoLocationController } from './geo-location.controller';
+import { GeoLocationService } from './geo-location.service';
+import { HttpModule } from '@nestjs/axios';
+import { ConfigModule } from '@nestjs/config';
+import { ClientsModule, Transport } from '@nestjs/microservices';
+import { getNatsOptions } from '@credebl/common/nats.config';
+import { CommonModule } from '@credebl/common';
+
+@Module({
+  imports: [
+    HttpModule,
+    ConfigModule.forRoot(),
+    ClientsModule.register([
+      {
+        name: 'NATS_CLIENT',
+        transport: Transport.NATS,
+        options: getNatsOptions(process.env.API_GATEWAY_NKEY_SEED)
```

Comment:
I think specific MS should use there own `NKEY SEED`

Reply:
Yes . I have added right NKEY_SEED.